run.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'lib/pascaltriangle'
+
+print "Please enter row numbers: "
+row_number = gets.chomp.to_i
```

Comment:
@zzokki81 please modify the program not to ask for the input, but to expect input from the cli.